tapfreighter/wallet.go

```diff
@@ -315,6 +315,11 @@ func createPassivePacket(params *address.ChainParams, passiveAsset *asset.Asset,
 	// asset.
 	outputAsset.SplitCommitmentRoot = nil
 
+	// If this was previously a time locked asset, it's now a simple asset.
+	// So we need to clear out the time locks as well.
+	outputAsset.LockTime = 0
```

Comment:
IIUC, wrt. tapchannels, this would cover the case of the input for the funding TX anchoring 2+ assets, where 1 asset is used for channel funding, and 1+ assets would be change (passive) and sent back to the funder in a separate output?

I think we don't cover that on the Terminal itests rn, will work on that.

Reply:
I think Oli tacked this on. 

At a glance, makes sense as the passive assets are always 1-in-1-out transactions so it doesn't go through the normal split creation case. 

Based on his comment, we can make a like `CopyFreshAssetTemplate` function that knows to blank out these fields, and we use it in both contexts. 

 > this would cover the case of the input for the funding TX anchoring 2+ assets, where 1 asset is used for channel funding, and 1+ assets would be change (passive) and sent back to the funder in a separate output?

Good question...today we still only support a single asset in the anchor. I think this is more precautionary for a case that can arise with custom vPSBT flows? 

cc @guggero